package.json

```diff
@@ -80,8 +81,9 @@
     "slugify": "^1.6.5",
     "swr": "^2.2.4",
     "uuidv7": "^0.6.3",
-    "viem": "^2.9.6",
-    "wagmi": "^2.5.17",
+    "viem": "2.9.6",
+    "wagmi": "2.5.17",
+    "webpack-bugsnag-plugins": "^1.8.0",
```

Comment:
I'm wondering if this can be a `devDependency` 🤔

Reply:
Good point, seems like it'll work that way too, so reinstalled it as a `devDependency`!